frontends/bsd/src/app_root.tsx

```diff
@@ -650,7 +653,7 @@ export function AppRoot({ card, hardware }: AppRootProps): JSX.Element {
                 <ScanButton
                   onPress={scanBatch}
                   disabled={isScanning}
-                  isScannerAttached={hasBatchScannerAttached}
+                  isScannerAttached={!!batchScanner}
```

Comment:
@eventualbuddha Should there be an eslint rule for this? I thought we preferred having a more explicit expression here like `batchScanner != undefined` but I'm forgetting if thats part of the code style guide or not.